_docs/tasks/security/health-check.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Enabling Istio CA health check
+overview: This task shows how to enable Istio CA health check.
+
+order: 70
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows how to enable Istio CA health check. Note this is an alpha feature after Istio V0.6.
+
+After Istio V0.6, Istio CA has a health check feature that can be optionally enabled.
```

Comment:
So this feature is disabled by default? If so, please mentioned it here.

Reply:
^After^Since

Reply:
Sure.